asserts/snapasserts/snapasserts.go

```diff
@@ -40,6 +41,39 @@ type Finder interface {
 	FindMany(assertionType *asserts.AssertionType, headers map[string]string) ([]asserts.Assertion, error)
 }
 
+// ValidationSetKey is a string-backed primary key for a validation set assertion.
+type ValidationSetKey string
```

Comment:
doesn't this also belong in validation_sets.go?  when I mentioned snapasserts yesterday I just meant the package (otherwise I hope I would have been explicit about .go, also it makes sense :) )

Reply:
You're right, that makes more sense. I've moved that and added some unit tests as well